pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -140,12 +143,18 @@ func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Contai
 		if !defaultHint.Preferred && bestHint.Preferred {
 			return fmt.Errorf("[memorymanager] failed to find the default preferred hint")
 		}
+
+		klog.V(4).InfoS("Topology Hint recomputed", "pod", klog.KObj(pod), "podUID", pod.UID, "containerName", container.Name, "previous", bestHint, "recomputed", defaultHint)
```

Comment:
I wonder if this should be log level 3 as well? Seems important enough to be at least log level 3 to me.

Reply:
I think you're right, fixing.